style.css

```diff
@@ -619,25 +570,208 @@ body {
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  margin-left: 50px;
-  margin-bottom: 20px;
 }
 
 .contact-text {
   font-style: normal;
-  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
   color: #ebebff;
-  margin-bottom: 40px;
-}
-
-.contact-form-background-image {
-  position: absolute;
-  width: 233px;
-  height: 670.17px;
-  left: 153px;
-  margin-left: -10px;
-  margin-top: 49px;
+  margin-right: 39px;
+  padding-right: 30px;
+  font-weight: 400;
+  width: 569px;
+  height: 102px;
+}
+
+/* ------------Desktop Version----------- */
+
+@media screen and (min-width: 768px) {
+  .body {
+    background: #f4f5f7;
+  }
+
+  .tool {
+    width: 1440px;
+    height: 816px;
+    top: 1px;
+  }
+
+  .toolbar {
+    display: flex;
+    margin-right: 100px;
+    padding-right: 100px;
+  }
+
+  .icons {
+    display: none;
+  }
+
+  .navu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .navul1 {
+    margin-left: 150px;
+    margin-top: -20px;
+  }
+
+  .navul2 {
+    margin-top: -20px;
+    margin-left: 840px;
+  }
+
+  .navulitems {
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-right: 100px;
+  }
+
+  .navli {
+    color: #344563;
+    margin: 5px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 40px;
+    background-image: url(./Headerbig.png);
+    margin-top: 50px;
+    border-bottom-left-radius: 150px;
+    border-top-left-radius: 150px;
+  }
+
+  .text1 {
+    flex-wrap: wrap;
+    width: 80%;
+    margin-left: 250px;
+    padding-top: 200px;
+  }
+
+  .text2 {
+    flex-wrap: wrap;
+    width: 60%;
+    margin-left: 250px;
+    margin-right: 20%;
+    padding-top: 100px;
+    margin-top: 5px;
+  }
+
+  .let {
+    margin-left: 250px;
+    margin-top: 5px;
+    padding-top: 50px;
+  }
+
+  .socialmediaicons {
+    padding-left: 250px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+    height: 496px;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    background-color: white;
+  }
+
+  .card__image {
+    width: 544px;
+    height: 448px;
+    left: 24px;
+    top: 24px;
+    border-radius: 8px;
+  }
+
+  .card__content {
+    width: 544px;
+    height: 465px;
+    left: 590px;
+    top: 24px;
+    padding: 24px;
+    gap: 24px;
+  }
```

Comment:
[OPTIONAL] The card alternates with the picture on the left and the other content on the right, whiles the next card have picture on the right and other content on the left.You can consider arranging your cards in that same manner to conform to the design. :+1: